lib/MusicBrainz/Server/Data/Tag.pm

```diff
@@ -91,6 +98,23 @@ sub get_cloud
             ),
         };
     })];
+}
+
+sub get_cloud
+{
+    my ($self, $genre_limit, $tag_limit) = @_;
+
+    my $cache = $self->c->cache('tag');
+    my $data = $cache->get('tag_cloud');
+    return $data if defined $data;
+
+    $genre_limit ||= 200;
+    $tag_limit ||= 50;
+
+    $data = {
```

Comment:
This'll cause an ISE when it's released on beta/production, since it uses the same cache key but changes the format of the data being stored there.  An easy way to avoid that would be to just change the cache key (genre_and_other_tag_clouds?).

Reply:
Just `'tag_cloud_data'` seems fine yet different, think I'll go with that